packages/calcite-components/src/components/list/list.tsx

```diff
@@ -103,6 +103,12 @@ export class List extends LitElement implements InteractiveComponent, SortableCo
   /** TODO: [MIGRATION] this flag was used to work around an issue with debounce using the last args passed when invoking the debounced fn, causing events to not emit */
   private willPerformFilter: boolean = false;
 
+  private containerRef: HTMLDivElement;
+
+  resizeObserver = createObserver("resize", () => {
```

Comment:
@Amretasre002762670 do we need the resize observer or would `height:100%` work in CSS?

I'm wondering if the resize observer is overkill and we could just set a fixed height or use height of 100%. cc @ashetland